plugins/embed/gui-themes/themes/zeit-amp/liveblog.css

```diff
@@ -0,0 +1,138 @@
+#livedesk-root { font-family: "TabletGothic", "Helvetica Neue", Helvetica, Arial, FreeSans, sans-serif; font-size: 1rem; line-height: 1.5; margin: 0 auto; max-width: 61.25em; }
```

Comment:
For AMP, all CSS must be inside a `<style amp-custom>` tag in the head of the document. I just develop the code here and then include the output in our CMS output.